src/transformers/models/albert/configuration_albert.py

```diff
@@ -165,3 +168,6 @@ def inputs(self) -> Mapping[str, Mapping[int, str]]:
                 ("token_type_ids", dynamic_axis),
             ]
         )
+
+
+__all__ = ["AlbertConfig", "AlbertOnnxConfig"]
```

Comment:
This could be removed in the future (v5?) if we add underscores to all "private" items that we don't want to export

Reply:
Down to do that in the futur

Reply:
I think we can also not break at all and have one dirty files that has like`AlbertConfig = _AlbertConfig` and so on and so forth (for all the stuff that should not be in the all)

Reply:
Addressed; keeping the `__all__` and removing some `@register` (now `@export`) instead.